lib/tasks/build.js

```diff
@@ -20,14 +20,17 @@ const log = require('../helpers/log');
 module.exports = function(gulp, config) {
 	let jsStream;
 	let sassStream;
+	const buildStream = merge();
 	if (typeof config.watching === 'undefined' || config.watching === 'js') {
-		jsStream = module.exports.js(gulp, config);
+		jsStream = module.exports.js(gulp, config)
+			.on('error', buildStream.emit.bind(buildStream, 'error'));
 	}
 	if (typeof config.watching === 'undefined' || config.watching === 'sass') {
-		sassStream = module.exports.sass(gulp, config);
+		sassStream = module.exports.sass(gulp, config)
+			.on('error', buildStream.emit.bind(buildStream, 'error'));
 	}
 	if (jsStream && sassStream) {
-		return merge(jsStream, sassStream);
+		return buildStream.add(jsStream, sassStream);
```

Comment:
If .add() modifies the stream in place and then returns it (I assume that's the case), rather than returning a new stream then it would be better written as:

```
buildStream.add(jsStream, sassStream);
return buildStream;
```